packages/core/src/Cardano/types/StakePool/StakePool.ts

```diff
@@ -86,4 +86,8 @@ export interface StakePool extends PoolParameters {
    * Sorted by epoch in ascending order.
    */
   epochRewards: StakePoolEpochRewards[];
+  /**
+   * Annual Percentage Yield (APY)
+   */
+  apy?: number;
```

Comment:
Not sure if this is the correct place for this

Reply:
I thought about it but I need the pool rewards to calculate this and the metrics are retrieved by another query. I would need to adjust the types but it's doable

Reply:
Yes, you are right, it was easier than I thought. Fixed it